nemo/collections/nlp/modules/common/prompt_encoder.py

```diff
@@ -73,13 +73,13 @@ def __init__(self, template: List[int], hidden_size: int, lstm_dropout: float, n
         )
 
     @typecheck()
-    def forward(self, enc_taskname) -> torch.Tensor:
-        input_embeds = self.embedding(self.seq_indices).unsqueeze(0)
-        if enc_taskname is not None:
```

Comment:
Yeah, I'm assuming a task name always needs to be given because after p-tuning is over, the virtual token embeddings will be moved to the prompt table, and the prompt table needs a task name to look up the correct embeddings.